k8s/jenkins/apptest/tester/tests/basic-suite.yaml

```diff
@@ -1,8 +1,250 @@
 actions:
-- name: dummy test
+- name: Jenkins service login
   bashTest:
     script: |-
-      exit 0
+      SERVICE="${APP_INSTANCE_NAME}-jenkins-ui"
+      IP=''
+      PORT=''
+      PASSWORD=''
+      FAILED=0
+
+      function wait_for_service_ip {
+        IP=$(kubectl get service "${SERVICE}" \
+          --namespace "${NAMESPACE}" \
+          -ojsonpath="{.spec.clusterIP}")
+        if [[ $? -ne 0 ]]; then
+          IP=""
+        fi
+      }
+
+      function wait_for_service_port {
```

Comment:
The same question. Isn't service port available at once?

Reply:
To keep consistency with the rest of the code.